props/saber_BC_buttons.h

```diff
@@ -1234,64 +2775,101 @@ case EVENTID(BUTTON_POWER, EVENT_FOURTH_HELD, MODE_OFF):
           PVLOG_VERBOSE << "The Force will be with you...always.\n";
         }
         return;
-      // Gesture on, bybass preon
+      case EFFECT_IGNITION:
+        scroll_presets_ = false;
+        saber_on_time_ = millis();
+        return;
+      // Gesture on, bypass preon
       case EFFECT_FAST_ON:
+        if (!SaberBase::IsOn()) FastOn();
 #ifdef ENABLE_FASTON
-        if (SFX_faston) hybrid_font.PlayCommon(&SFX_faston);
+        if (SFX_faston) hybrid_font.PlayPolyphonic(&SFX_faston);
 #endif
         scroll_presets_ = false;
         return;
       case EFFECT_FAST_OFF:
-        if (SaberBase::IsOn()) {
-          Off(OFF_FAST);
-          saber_off_time_ = millis();
-        }
+        if (on_pending_) return;
+        Off(OFF_FAST);
```

Comment:
This is super weird.
This means that you can turn off the saber from a style with TrDoEffect<>
Is that intentional?
Why is Off() here?

Reply:
Hmm. I have no recollection, it's been there a long while.
I mean, being able to turn off the saber from a style with TrDoEffect<> isn't necessarily bad I suppose?
But no, it was not intentional for that purpose.

wow - Looking at it now, I have both FAST_ON and FAST_OFF in SB_Effect. I think I had once assumed they were triggered by saberbase and therefore were a place to set variables etc...
Not right, not good.
So effectively, all of this can go away, right?
```cpp
      // Gesture on, bypass preon
      case EFFECT_FAST_ON:
        if (!SaberBase::IsOn()) FastOn();
#ifdef ENABLE_FASTON
        if (SFX_faston) hybrid_font.PlayPolyphonic(&SFX_faston);
#endif
        scroll_presets_ = false;
        return;
      case EFFECT_FAST_OFF:
        if (on_pending_) return;
        Off(OFF_FAST);
        saber_off_time_ = millis();
        battle_mode_ = false;
        spam_blast_ = false;
        muted_ = false;
        return;
        ```
        
I do not remember the story with faston/fastout really so just removing it all seems like the right thing, especially if it wasn't doing anything anyway.

I tested without this stuff and no difference that I see, so...
removed.